tests/unit/test_endpoint.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2012-2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2012-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This was part of a larger refactoring, which I rebased.  Looks like I didn't quite get them split cleanly, I'll change this back.